packages/deb/build.go

```diff
@@ -131,7 +132,7 @@ func (c cfg) run() error {
 	var w []work
 
 	srcdir := filepath.Join(c.DistroName, c.Package)
-	dstdir, err := ioutil.TempDir(os.TempDir(), "debs")
+	dstdir, err := ioutil.TempDir("", "debs")
```

Comment:
@saschagrunert should we use https://golang.org/pkg/os/#MkdirTemp?

Reply:
Yes, fixed as suggested :+1: